<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorAutocompletionQScintillaPage</class>
 <widget class="QWidget" name="EditorAutocompletionQScintillaPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>257</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>&lt;b&gt;Configure QScintilla Completion&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="Line" name="line6">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="acShowSingleCheckBox">
     <property name="toolTip">
      <string>Select this, if single entries shall be inserted automatically</string>
     </property>
     <property name="text">
      <string>Show single</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="acFillupsCheckBox">
     <property name="toolTip">
      <string>Select to enable the use of fill-up characters to autocomplete the current word</string>
     </property>
     <property name="whatsThis">
      <string>&lt;b&gt;Use fill-up characters&lt;/b&gt;&lt;p&gt;Select to enable the use of fill-up characters to autocomplete the current word. A fill-up character is one that, when entered while an auto-completion list is being displayed, causes the currently selected item from the list to be added to the text followed by the fill-up character.&lt;/p&gt;</string>
     </property>
     <property name="text">
      <string>Use fill-up characters</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Source</string>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QRadioButton" name="acSourceDocumentRadioButton">
        <property name="toolTip">
         <string>Select this to get autocompletion from current document</string>
        </property>
        <property name="text">
         <string>from Document</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="acSourceAPIsRadioButton">
        <property name="toolTip">
         <string>Select this to get autocompletion from installed APIs</string>
        </property>
        <property name="text">
         <string>from API files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="acSourceAllRadioButton">
        <property name="toolTip">
         <string>Select this to get autocompletion from current document and installed APIs</string>
        </property>
        <property name="text">
         <string>from Document and API files</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>456</width>
       <height>51</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>acShowSingleCheckBox</tabstop>
  <tabstop>acFillupsCheckBox</tabstop>
  <tabstop>acSourceDocumentRadioButton</tabstop>
  <tabstop>acSourceAPIsRadioButton</tabstop>
  <tabstop>acSourceAllRadioButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
